### PR TITLE
Production changes

### PR DESCRIPTION
**Pull request by dgarciabriseno**

### CODE REVIEW

File: README.txt [hunk 1]
```diff
@@ -39,28 +39,11 @@ following software:
 INSTALLATION:
 
 Instructions for installing Helioviewer.org are available online at
-http://wiki.helioviewer.org/wiki/Installing_Helioviewer.org
+http://wiki.helioviewer.org/wiki/index.php?title=Installing_Helioviewer.org
```
Comment (dgarciabriseno) [inline README.txt]: We should fix these links, I'll update them with the github.io pages.

File: README.txt [hunk 2]
```diff
@@ -39,28 +39,11 @@ following software:
...
-
-Helioviewer.org acknowledges the support of the following instrument teams,
-mission teams and institutions - SDO(NASA), SOHO(ESA/NASA), STEREO(NASA),
-PROBA2(ESA), Yohkoh(JAXA/NASA/PPARC), NASA GSFC, Royal Observatory of
-Belgium, LMSAL, SDAC, Stanford University, Montana State University/FFT
-
-Helioviewer.org acknowledges the use of the following tools:
- - www.browserstack.com, for testing the web application on multiple browsers
-   and OSs,
- - www.highcharts.com, for interactive JavaScript charts.
```
Comment (dgarciabriseno) [inline README.txt:66]: Do we want to keep these credits? I think it's good practice/sportsmanship.

**Comment by dgarciabriseno** [comment]
Remove empty files:
 docs/v2/version_specific.css
 docs/v2/version_specific.js

File: README.txt (hunk 1 above)
Comment (wafels) [reply README.txt]: This one, right? https://helioviewer-project.github.io/install/

File: README.txt (hunk 2 above)
Comment (wafels) [reply README.txt:66]: Yes we should. Highcharts is used in the timeline functionality.

File: README.txt (hunk 1 above)
Comment (dgarciabriseno) [reply README.txt]: yep

**Review by wafels — APPROVED** [review]
Looks ok to me.

**Pull request merged**